xbmc/addons/kodi-addon-dev-kit/include/kodi/xbmc_pvr_types.h

```diff
@@ -402,6 +402,7 @@ extern "C" {
     unsigned int iEncryptionSystem;                                    /*!< @brief (optional) the encryption ID or CaID of this channel */
     char         strIconPath[PVR_ADDON_URL_STRING_LENGTH];             /*!< @brief (optional) path to the channel icon (if present) */
     bool         bIsHidden;                                            /*!< @brief (optional) true if this channel is marked as hidden */
+    bool         bHasArchive;                                          /*!< @brief (optional) true if this channel supports archive */
```

Comment:
Could you please elaborate here what an archive is in this context.

Reply:
Ok, will do. I think the way you put it "server-side back buffer" is probably a good description.
Maybe the whole "archive" part should be changed to something else? I couldn't come up with a better name for it than this, but I'm open to suggestions.